stories/CssVariables.stories.js

```diff
@@ -7,13 +7,13 @@ const App = ({ children }) => <div>{children}</div>
 
 storiesOf('CssVariables', module).add('Default', () => (
     <App>
-        <CssVariables />
+        <CssVariables variables={{ 'color-red': '#a83c32' }} />
```

Comment:
Can we use a name here that doesn't look like you'd be overwriting a theme value?
Right now it looks like the intended use-case is do customize the theme while that's exactly what we don't want

Reply:
Yeah no problem. Though they wouldn't be overriding the theme. The internals of our components aren't affected by anything you set here. I think Viktor's concerns were on using css-variables internally which users would then be able to override.

And if users want to import the theme and then override it there's not much you can do about that.